scripts/smoketest.py

```diff
@@ -1,56 +1,101 @@
         # Note(austin) The two inference calls will hang on mac with unsupported hardware error
         # Skip these with SKIP_INFERENCE_TESTS=true make docker-test
-        pytest.param("layout-parser-paper.pdf", "application/pdf", marks=pytest.mark.skipif(
+        pytest.param("layout-parser-paper.pdf", "application/pdf", "application/json", "fast", marks=pytest.mark.skipif(
             skip_inference_tests, reason="emulated architecture")
         ),
-        pytest.param("layout-parser-paper-fast.jpg", "image/jpeg", marks=pytest.mark.skipif(
+        pytest.param("layout-parser-paper-fast.jpg", "image/jpeg", "application/json", "fast", marks=pytest.mark.skipif(
             skip_inference_tests, reason="emulated architecture")
-        )
+        ),
+        ("fake-text.txt", "text/plain", "text/csv", "fast"),
+        ("announcement.eml", "message/rfc822", "text/csv", "fast"),
+        ("fake-email-attachment.eml", "message/rfc822", "text/csv", "fast"),
+        ("fake-email-image-embedded.eml", "message/rfc822", "text/csv", "fast"),
+        ("fake-email.eml", "message/rfc822", "text/csv", "fast"),
+        ("fake-html.html", "text/html", "text/csv", "fast"),
+        ("fake-power-point.ppt", "application/vnd.openxmlformats-officedocument.presentationml.presentation", "text/csv", "fast"),
+        ("fake.doc", "application/msword", "text/csv", "fast"),
+        ("fake.docx", "application/vnd.openxmlformats-officedocument.wordprocessingml.document", "text/csv", "fast"),
+        ("family-day.eml", "message/rfc822", "text/csv", "fast"),
+        pytest.param("fake-excel.xlsx", None, "text/csv", "fast", marks=pytest.mark.xfail(reason="not supported yet")),
+        pytest.param("layout-parser-paper.pdf", "application/pdf", "text/csv", "hi_res", marks=pytest.mark.skipif(
+            skip_inference_tests, reason="emulated architecture")
+        ),
+        pytest.param("layout-parser-paper-fast.jpg", "image/jpeg", "text/csv", "hi_res", marks=pytest.mark.skipif(
+            skip_inference_tests, reason="emulated architecture")
+        ),
     ]
 )
-def test_happy_path(example_filename, content_type):
+def test_happy_path(example_filename, content_type, output_format, strategy):
     """
     For the files in sample-docs, verify that we get a 200
     and some structured response
     """
     test_file = Path("sample-docs") / example_filename
-    response = send_document(test_file, content_type)
+    response = send_document(
+        filename=test_file,
+        strategy=strategy,
+        content_type=content_type,
+        output_format=output_format,
+    )
 
     assert(response.status_code == 200)
     assert len(response.json()) > 0
-    assert len("".join(elem["text"] for elem in response.json())) > 20
+    # NOTE(kravetsmic): looks like a bug on macOS (m1), incorrectly scanned text from images and pdf files
```

Comment:
please say more. is it all images and pdf's or are there specific files causing a problem? this may be worth creating an issue for in the unstructured repo.